docs/filtering_configuration.md

```diff
@@ -99,3 +99,36 @@ releases =
 ```
 
 Note the same `filter_regex` section may include a `packages` and a `releases` entry with any number of regular expressions.
+
+
+### Platform-specific binaries filtering
+
+This filter allows advanced users not interesting in Windows/macOS/Linux specific binaries to not mirror the corresponding files.
+
+
+``` ini
+[blacklist]
+plugins =
+    exclude_platform
+platforms =
+    windows
+```
+
+Avalaible platforms are: `windows` `macos` `freebsd` `linux`.
```

Comment:
Should we link to and support all https://packaging.python.org/specifications/platform-compatibility-tags/ too?

Reply:
response below